qemu/tests/nvdimm.py

```diff
@@ -155,6 +155,9 @@ def run(test, params, env):
             time.sleep(10)
             mems += target_mems
         error_context.context("Verify nvdimm in monitor and guest", test.log.info)
+        pkgs = params["depends_pkgs"].split()
```

Comment:
Not all case have param `depends_pkgs`, such as case `nvdimm_emulate` will introduce `KeyError`.
Suggest  `params.object("depends_pks")`

Reply:
typo,   `objects`

Reply:
Updated and could you help to review it again @yanan-fu  Thanks